src/main/java/com/aws/greengrass/deployment/DeploymentConfigMerger.java

```diff
@@ -147,6 +156,26 @@ private void updateActionForDeployment(Map<String, Object> newConfig, Deployment
             return;
         }
 
+        // As long as the timeout is not 0, we will try to run the connectivity validation
+        int timeout = 0;
+        DeploymentConfigurationValidationPolicy policy = deployment.getDeploymentDocumentObj()
+                .getConfigurationValidationPolicy();
+        if (policy != null) {
+            timeout = Coerce.toInt(policy.timeoutInSeconds());
+        }
+        long timestamp = deployment.getDeploymentDocumentObj().getTimestamp();
```

Comment:
if the scope of `timestamp` is limited to within the try block then you should not declare this outside that block